src/%ZPM/PackageManager/Client/Settings.cls

```diff
@@ -22,7 +22,19 @@ Parameter TerminalPrompt = "TerminalPrompt";
 
 Parameter PublishTimeout = "publish_timeout";
 
-Parameter CONFIGURABLE = "trackingId,analytics,ColorScheme,TerminalPrompt,PublishTimeout";
+/// A path pointing either to a python executable or a pip executable, controlled by #UseStandalonePip
+/// In the special case where the path is empty, the script tries to resolve the path in the following order:
+/// 1. Where available (typically 2024.2+) - Use PythonRuntimeLibrary in iris.cpf to find the directory containing the python executable
+/// 2. On Windows, try to find <iris-root>/bin/irispip.exe
+/// 3. Unless UseStandalonePip is explicitly set to 1, try to find whichever python3/python is first available in $PATH
+/// 4. Unless UseStandalonePip is explicitly set to 0, try to find whichever pip3/pip is first available in $PATH
+Parameter PipCaller = "PipCaller";
+
+/// Possible values: "", 0, 1
+/// Indicates whether PipCaller is a pip executable instead of python
+Parameter UseStandalonePip = "UseStandalonePip";
```

Comment:
Should this be 0/1 and not a string? Also, can set types to parameters of BOOLEAN here and STRING above.

Reply:
So I am under the impression that all parameters here are meant to be used as the index to ^zpm.settings. 
For example, in this method
https://github.com/intersystems/ipm/blob/5a4e85ff129573e6337ab205fc07946859bf3e33/src/%25ZPM/PackageManager/Client/Settings.cls#L136-L139
The usage of `#..trackingId` suggests that the parameter `trackingId` is the key.

Reply:
Clarifying, the type of the parameter is always a string. The values of the parameters are "PipCaller" and "UseStandalonePip" respectively, both strings.